eng/common/scripts/Download-Latest-Contract.ps1

```diff
@@ -0,0 +1,78 @@
+[CmdletBinding()]

```

Comment:
Can we add documentation on how language teams are supposed to integrate with this file?

Touch on areas like
- what if you have no ga releases yet
- which functions in do you need to override in your language-settings.ps1
- what return value you are supposed to expect and what to do with empty string
- where the latest contract will end up
- etc